### PR TITLE
Add database automation

### PR DESCRIPTION
**Pull request by juicemia**
This adds a basic compose file with a db service that created the appropriate database and configures it for use. Currently dev only.

### CODE REVIEW

File: .dev/docker/db/docker-entrypoint-initdb.d/setup.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE gcon_development;
```
Comment (briandowns) [inline .dev/docker/db/docker-entrypoint-initdb.d/setup.sql:1]: Will the application know to use that database when it spins up?

Comment (juicemia) [reply .dev/docker/db/docker-entrypoint-initdb.d/setup.sql:1]: That's the name of the DB that the app is configured to use when run in dev mode right now. I don't know how we're getting this deployed to production yet exactly so I didn't want to spin my wheels adding configuration stuff that might need to be scrapped later.

Comment (briandowns) [reply .dev/docker/db/docker-entrypoint-initdb.d/setup.sql:1]: Sounds good. Thought I'd ask. :D

**Review by briandowns — DISMISSED** [review]

**Comment by briandowns** [comment]
Can you update this branch before merging?

**Review by briandowns — APPROVED** [review]

**Pull request merged**